docs/platforms-release-process.md

```diff
@@ -118,114 +205,147 @@ If you have prepared the release notes in your release branch for a patch releas
 
 For iOS, you may have to cherry-pick the commit for `Added X.Y.Z to CDVAvailability.h (via coho)` into the master branch as well.
 
----
+### Special Case 2: releasing new commits from an already-existing release branch
 
 If you are releasing new commits from an already-existing release branch, remember to merge in or cherry-pick relevant commits from master into the release branch!
 
-After, prepare your release branch by using `coho prepare-release-branch` command, which handles the following steps:
- * Updating `cordova.js` snapshot
- * Creating a release branch (if it doesn't already exist)
- * Updating version numbers (`VERSION` file & package.json). On `master`, it gives version a minor bump and adds `-dev`
+### Create Release Branch
+
+Create and prepare your release branch by using `coho prepare-release-branch` command, which handles the following steps:
 
+1. Creates a release branch `5.0.x` (if it doesn't already exist)
+2. Updates `cordova.js` snapshot on both `5.0.x` and `master`
+3. Updates version numbers (`VERSION` files) on `5.0.x`
```

Comment:
It also updates in other files but that is platform specific. For example, it updates build.gradle version for android. It does similar stuff for iOS.

Reply:
Reworded that and added your example.